packages/better-auth/src/plugins/organization/types.ts

```diff
@@ -447,7 +452,7 @@ export interface OrganizationOptions {
 				[key: string]: any;
 			};
 			user: User & Record<string, any>;
-			member: Member & Record<string, any>;
+			member?: Member & Record<string, any>;
```

Comment:
`member` needs to become optional in these hooks, but I was wondering if we'd want to pass in whether the user performing the update is a super admin.